frontend/test/revisions/revisions.fixtures.ts

```diff
@@ -27,3 +27,17 @@ export const revisionsMockFactory = (overrides?: Partial<IRevision>): IRevision
 	void: faker.datatype.boolean(),
 	...overrides,
 });
+
+export const mockCreateRevisionBody = (overrides?: Partial<CreateRevisionBody>): CreateRevisionBody => ({
+	revisionTag: faker.random.words(1),
+	revisionDesc: faker.random.words(3),
+	file: new File(['file'], 'filename.obj'),
+	importAnimations: false,
+	timezone: 'Europe/London',
+	containerName: faker.random.words(1),
+	containerType: 'Other',
+	containerUnit: 'cm',
+	containerDesc: faker.random.words(3),
+	containerCode: faker.random.words(1),
+	...overrides,
+});
```

Comment:
Are you missing the new line at the end of the file?

Reply:
Yes. I think eslint doesn't touch the test files so you can often see linting errors there